Project.toml

```diff
@@ -61,6 +62,7 @@ ReverseDiff = "1"
 SciMLBase = "1.79.0, 2"
```

Comment:
Doesn't this need to be bumped?

Reply:
Why would it? Nothing in Optimization changed that prevents it from being compatible with previous versions of SciMLBase.

Reply:
oh I thought the RAT v3 or SII v0.3 changes would require it, but if that back compatibility is kept then we're good.